spec/models/application_spec.rb

```diff
@@ -59,10 +59,9 @@
 
     it 'removes the flag if the value is 0' do
       flag = flags.sample.to_sym
-      subject.flags = [flag]
```

Comment:
I'm not too sure on this fella, either. This meta-programming loop in `Application` is test covered according to https://codeclimate.com/github/rails-girls-summer-of-code/rgsoc-teams/Application, but it also has a ternary op in it so I don't know of both branches are really tested.

The elements of the `flags` array now being strings makes sense: I don't think AR serializes symbols. We just need to investigate why it would still return symbols occasionally then.

As for the assignment in L62: I don't know if it's part of the test (ie. checking whether assigning the full array instead of individual values through the meta-programmed quasi-boolean accessors) or just setting up the world for the test. I will check it our locally and play with it myself a little.